app/views/stations/show.erb

```diff
@@ -0,0 +1,11 @@
+<a href='/stations'>Back to All Stations</a>
+<h1>Viewing Station: <%=@station.name%></h1>
+  <p>Dock Count: <%= @station.dock_count%></p>
+  <p>City: <%= @station.city%></p>
+  <p>Date Installed: <%= @station.installation_date.strftime("%m/%d/%Y")%></p>
+  <form class="" action="/stations/<%= @station.id %>/edit" method="put">
```

Comment:
Same comment here regarding forms with `put`. See docs [here](https://www.w3.org/TR/html401/interact/forms.html#h-17.13.1) for more info.